hw/dv/dpi/usbdpi/monitor_usb.c

```diff
@@ -101,31 +101,31 @@ void monitor_usb(void *mon_void, FILE *mon_file, int loglevel, int tick,
     if (hdrive) {
       fprintf(mon_file, "mon: %8d: Bus clash\n", tick);
     }
-    if (d2p & D2P_TXMODE_SE) {
-      dp = ((d2p & D2P_DP_EN) && (d2p & D2P_DP)) ? 1 : 0;
-      dn = ((d2p & D2P_DN_EN) && (d2p & D2P_DN)) ? 1 : 0;
-    } else {
+    if (d2p & D2P_SE0_EN) {
       if ((d2p & D2P_SE0) || !(d2p & D2P_D_EN)) {
         dp = 0;
         dn = 0;
       } else {
         dp = (d2p & D2P_D) ? 1 : 0;
         dn = (d2p & D2P_D) ? 0 : 1;
       }
+    } else {
+      dp = ((d2p & D2P_DP_EN) && (d2p & D2P_DP)) ? 1 : 0;
+      dn = ((d2p & D2P_DN_EN) && (d2p & D2P_DN)) ? 1 : 0;
     }
     mon->driver = M_DEVICE;
   } else if (hdrive) {
-    if (d2p & D2P_TXMODE_SE) {
```

Comment:
This must've been an oops before: The TX mode only describes how the device drives to the host, and it has no relationship to how the host drives to the device.